src/flags/orgFlags.ts

```diff
@@ -10,7 +10,14 @@ import { Messages, Org, ConfigAggregator } from '@salesforce/core';
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/sf-plugins-core', 'messages');
 
-const maybeGetOrg = async (input?: string): Promise<Org | undefined> => Org.create({ aliasOrUsername: input });
+const maybeGetOrg = async (input?: string): Promise<Org | undefined> => {
+  try {
+    return Org.create({ aliasOrUsername: input });
+  } catch (e) {
```

Comment:
The catch should be checking to see if the user provided a value. If so, the the error should be re-thrown.